src/api.js

```diff
@@ -0,0 +1,7 @@
+export function fetchAllArticles() {
+  return fetch(
```

Comment:
Nice work extracting logic out to a separate file - for later tickets I'd recommend refactoring to use axios, it makes post/patch/delete requests a lot easier!